.travis.yml

```diff
@@ -14,6 +14,6 @@ install:
     - python setup.py install
 script:
     - nosetests . --with-coverage
-    - flake8 --count pyglmnet --exit-zero;
+    - flake8 --count pyglmnet
```

Comment:
yes I figured ;) I guess it's a trade-off. If your code is easy to read (due to the conventions) and understand, then you will attract good developers. But then they will discover that the reason it's clean is partly due to the conventions :)